test/base64_unittest.cc

```diff
@@ -0,0 +1,38 @@
+#include "../src/base64.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+TEST(EncodeTest, EmptyEncode) {
+  const std::string empty_str("");
+  EXPECT_EQ(
+      "",
+      base64::Encode(empty_str)
+  );
+}
+
+TEST(EncodeTest, SimpleEncode) {
+  const std::string simple_str("tes");
+  EXPECT_EQ(
+      "dGVz",
+      base64::Encode(simple_str)
+  );
+}
+
+TEST(EncodeTest, NoPaddingEncode) {
```

Comment:
Do you want to add a comment explaining the "no padding" bit? E.g., `// 5 bytes should produce 7 characters`.

Reply:
Yep, done.

Reply:
I've been trying to come up with a good name for the "padding" tests... The best I got so far is "phantom" (as in "phantom characters at the end"). So this test would have been named "OnePhantom", and the double padding one below would become "TwoPhantoms". WDYT? Happy to hear suggestions for better names. Ultimately, I think we ought to explain what we're testing in a comment if the test name isn't sufficiently expressive.
Your comment explains "no padding", but not the difference between the single and double pad characters.

Reply:
Phantom sounds good to me.  Hopefully the new comment is more clear.